src/openzeppelin/utils/constants.cairo

```diff
@@ -20,7 +20,7 @@ const IERC721_RECEIVER_ID: felt252 =
 // AccessControl
 // See: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/IAccessControl.sol
 const IACCESSCONTROL_ID: felt252 =
-    0x262988e98e77072d75a776162d7e30e5fe024a71909613481916261961df736;
+    0x23700be02858dbe2ac4dc9c9f66d0b6b0ed81ec7f970ca6844500a56ff61751;
```

Comment:
i don't think having this constant duplicated here and in `access/accesscontrol.cairo` is a good idea. let's keep just one.

Reply:
Your suggestion is just for AccessControl? All these constants are duplicated, and not a single one is being used from the constant module. Should I remove just AccessControl, or are you suggesting getting rid of the constants module? (at least current contents)

Reply:
if all constants are already available from their respective modules and no single reference to this module exists (not even in tests) then we should remove it

Reply:
There is no single reference indeed, but maybe it is better to let this for a separate PR? Even when it is a fairly small change, it will delay things. For example, should we add reference comments in the modules? that's not duplicated.

I can open a different issue and submit the PR removing the constants module if you agree.